docker-compose.yml

```diff
@@ -2,8 +2,14 @@ version: '3'
 services:
   rgit:
     image: ghcr.io/w4/rgit:main
+    command:
+      - "[::]:8000"
+      - /git
+      - -d /tmp/rgit-cache.db
     volumes:
-      - /path/to/my-repos:/git
+      - /volume/git:/git
```

Comment:
Nah

Reply:
It's going to be different for each environment, so the difference between the two paths is completely inconsequential. `/volume` doesn't exist in a standard environment, and either does `/path/to/my-repos`, so there's no danger of exposing anything here.

I don't understand your gripe.